service/controller/v26/resource/configmap/resource.go

```diff
@@ -83,7 +96,29 @@ func containsConfigMap(list []*apiv1.ConfigMap, item *apiv1.ConfigMap) bool {
 	return true
 }
 
-func getConfigMapByName(list []*apiv1.ConfigMap, name string) (*apiv1.ConfigMap, error) {
+// equals asseses the equality of ConfigMaps with regards to distinguishing
+// fields.
+func equals(a, b *corev1.ConfigMap) bool {
```

Comment:
This does also not look like labels changing.

Reply:
To update labels as operator desired, we need to compare label values first between current and desired configmaps, so I need to this function to compare it, does it make sense to you, WDYT?